Updating header with new Crayons buttons (#15653)
forem/forem: app/javascript/crayons/CTAs/__stories__/CTAs.mdx

## Usage

> Under the hood, CTA outputs `<a>` element, therefore standard link markup should be used,
> including an `href` (links omitting `href` attributes are not focusable by keyboard).

Reviewer: nitpick (non-blocking): wording

```suggestion
A CTA renders an anchor element, therefore standard link markup must be used,
including a `href` attribute, otherwise, it is not focusable by the keyboard.
```

#### Icon-only

> Call To Actions should never come without label, therefore it is forbidden to create CTAs with icon only.
> For such a scenario you should use `<Link />s` or `<Button />s` (depending on use case).

Reviewer: nitpick (blocking): Call To Actions shouldn't be capitalized for each word. Also suggesting some wording.

```suggestion
A call to action requires a label, therefore it is forbidden to create a CTA with an icon only.
For such a scenario, use a `<Link />` or `<Button />` component (depending on the use case).
```
